liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java

```diff
@@ -235,7 +236,14 @@ public boolean isReservedWord(String tableName) {
 
     @Override
     protected String getConnectionSchemaName() {
-        return "public";
+        try {
+            String currentSchema = ExecutorService.getInstance().getExecutor(this)
+                    .queryForObject(new RawCallStatement("select current_schema"), String.class);
```

Comment:
Hi, 
This query should be: "select current_schema()"
Please see: http://www.postgresql.org/docs/8.3/static/functions-info.html
Can you please fix this ?

Thanks,
Daniel